features/storage/blockdevice/VerifyReadBlockDevice.cpp

```diff
@@ -0,0 +1,129 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017-2017 ARM Limited
+ *
```

Comment:
I _think_ this is missing the SPDX identifier.

Reply:
You're right. Copy pasted an old file. Fixed now.

Reply:
The text still looks odd compare to the original license, can you update (check drivers/platform files)